code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -98,6 +98,33 @@
 			user << "<span class='warning'>You need a working light!</span>"
 			return
 
+	if(istype(W, /obj/item/weapon/storage/box/lights))
+		var/obj/item/weapon/storage/box/lights/B = W
+		var/tmp/replacement_status = 1 // 0-useable lights found, 1-no useable lights, 2-light replacer was full

```

Comment:
the ```/tmp``` modifier does nothing here.
it's used for vars not saved into savefiles, and you're not on a class OR saving things here.